autoload/test/strategy.vim

```diff
@@ -106,6 +113,9 @@ function! test#strategy#neovim_sticky(cmd) abort
     if get(g:, 'test#neovim_sticky#kill_previous', 0)
       let l:cmd = [""] + l:cmd
     endif
+    if get(g:, 'test#neovim_sticky#reopen_window', 0)
```

Comment:
There is a visibility check with `win_findbuf` below already, maybe it would make more sense to put it there?